apps/site/src/views/Resources/sections/ApiResources/ApiResources.tsx

```diff
@@ -10,9 +10,9 @@ import ApiResourcesList from "./config";
 function ApiResources() {
 	return (
 		<Container>
-			<Row className={styles.row}>
+			<Row>
 				{/* Card Component */}
-				<Col className={styles.column} xl={12}>
+				<Col className={styles.column}>
```

Comment:
Thought: at this point with only one row element, do we even need the column and row anymore?